glue_jupyter/table/table.vue

```diff
@@ -15,7 +15,7 @@
       <template v-slot:header="props">
         <thead>
           <tr>
-            <th style="padding: 0 10px; width: 40px">#</th>
+            <th :style="'padding: 0 10px; width: '+Math.ceil(Math.log10(total_length))*20+'px'">#</th>
```

Comment:
```suggestion
            <th :style="'padding: 0 10px; width: '+Math.max(1, Math.ceil(Math.log10(total_length)))*20+'px'">#</th>
```
Catching case `1` is a good idea indeed.